src/registry_schemas/schemas/amalgamation_application.json

```diff
@@ -14,7 +14,6 @@
                 "type",
                 "amalgamatingBusinesses",
                 "nameRequest",
-                "offices",
```

Comment:
Even though the Filer will not look at this (or shareStructure) for a short-form amalgamation (as it will copy them from the holding/primary business), the UI will still provide these in the filing JSON.

Would it make sense to keep offices and shareStructure here for regular amalgamations?

Reply:
Oh, I see what you did below -- these are required for a regular amalg.

Well then, do you prefer what you've done here, or just leave it the way it was? Up to you.

Reply:
Just wanted to make the schema more meaningful as we may have api users planning to use it in the future